var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -645,6 +678,9 @@ def common_config_args(self):
         if qtplat is not None:
             config_args.extend(["-platform", qtplat])
 
+        if not IS_WINDOWS:
+            config_args.extend(["-mp", "-icu"])
```

Comment:
(question) I don't see these args being added before this PR, which is about building `qt` on Windows; this specific logic only affects `qt` builds on other systems.

Reply:
Hmm, I guess I assumed when Dan disabled the deps to focus on just getting Qt with bundled deps to build that he had taken this out, and put it back based on the Qt docs. I'll remove it.